src/utilities/Helpers.ts

```diff
@@ -1,5 +1,25 @@
 import dayjs from "dayjs";
+import dateFormatter from "dayjs";
+import utc from "dayjs/plugin/utc";
+import timezone from "dayjs/plugin/timezone";
+import customParseFormat from "dayjs/plugin/customParseFormat";
 
-export default function subtractYears(numOfYears: any, date: Date = new Date()) {
+dateFormatter.extend(utc);
+dateFormatter.extend(timezone);
+dateFormatter.extend(customParseFormat);
+
+export default function subtractYears(numOfYears: number, date: Date = new Date()): Date {
     return dayjs(date).subtract(numOfYears, "year").toDate();
 }
+
+export function mirDateFormat(delimeter = "/"): string {
+    return `DD${delimeter}MM${delimeter}YYYY`;
+}
+
+export function mirBstDateFormatter(date: Date | string, includeTime = false): string {
```

Comment:
The `includeTime` parameter leads to confusing code where it is called. i.e. when I read `mirBstDateFormatter(date, true)` I have no idea what `true` means.

I think I'd suggest a "date formatter" function and a "datetime formatter" function.